include/oi/types/st.h

```diff
@@ -16,10 +16,52 @@
 #ifndef OI_TYPES_ST_H
 #define OI_TYPES_ST_H 1
 
-namespace ObjectIntrospection {
-namespace types {
-namespace st {
+/*
+ * Static Types
+ *
+ * OI employs a data segment to transfer information about the probed object to
+ * the debugger. Static Types are used with the `-ftyped-data-segment` feature
+ * to provide a compile time description of the contents of this data segment.
+ *
+ * DataBuffer represents any type with two methods: `void write_byte(uint8_t)`,
+ * which writes a given byte to the buffer; and, `size_t offset()`, which
+ returns
+ * the number of bytes written. The only implementation at present is
+ DataSegment,
+ * which is defined in `FuncGen::DefineDataSegmentDataBuffer`. Each Static Type
```

Comment:
```suggestion
 * returns the number of bytes written. The only implementation at present is
 * DataSegment, which is defined in `FuncGen::DefineDataSegmentDataBuffer`. Each
 * Static Type ...
```

A classic formatting nitpick ;)

I'm guessing the linter wrapped your lines poorly. This has happened in a few lines in this block comment, not just here.